src/schemas/actions.json

```
@@ -65,7 +65,7 @@
             "$id": "#/actions/items/anyOf/0/properties/postDownload",
             "type": "array",
             "title": "Post Download",
-            "description": "Remote files to download when the Action is complete.",
+            "description": "Remote files/folders to download when the Action is complete. If items ends with `/`, a directory is assumed.",
```

It may be not worth keeping the last part since it's automatically detected now!
```suggestion
            "description": "Remote files/folders to download when the Action is complete.",
```